internal/core/network/network.go

```diff
@@ -21,15 +37,15 @@ func Get(ctx context.Context, id string) (types.NetworkResource, error) {
 	defer cli.Close()
 
 	filters := filters.NewArgs()
-	filters.Add("id", id)
+	filters.Add(string(f), value)
```

Comment:
Do we need the custom type just to cast here?

Reply:
You got me 😊 
Not necessary!
I'll modify you're right 😄